lib/basedriver/capabilities.js

```diff
@@ -76,7 +76,7 @@ function stripAppiumPrefixes (caps) {
   const prefixedCaps = _.filter(_.keys(caps), cap => `${cap}`.startsWith(prefix));
   const badPrefixedCaps = [];
   const unprefixedCaps = _.filter(_.keys(caps), (cap) => (
-    `${cap}`.includes(':') && !isStandardCap(cap)
```

Comment:
I don't think `_.keys` will ever return anything but an array of strings (it relies on the logic of http://ecma-international.org/ecma-262/7.0/#sec-enumerableownnames) so the coercing it to string is not needed.